### PR TITLE
[Executor/View] Add Executor View

### PR DESCRIPTION
**Pull request by jyoungyun**
This commit adds executor view on ONE view container.

ONE-vscode-DCO-1.0-Signed-off-by: Jiyoung Yun <jy910.yun@samsung.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,6 +37,10 @@
           "id": "CompilerToolchainView",
           "name": "Compiler Toolchain",
           "contextualTitle": "ONE"
```
Comment (jyoungyun) [inline package.json:39]: Related pr: #472

**Review by yunjayh — APPROVED** [review]
LGTM

**Comment by jyoungyun** [comment]
I rebased this PR. Please check this code again. /cc @yunjayh @dayo09

**Review by yunjayh — APPROVED** [review]

**Comment by jyoungyun** [comment]
PTAL /cc @dayo09

**Review by dayo09 — APPROVED** [review]

**Pull request merged**